pkg/larker/testdata/simple-task/expected.yaml

```diff
@@ -0,0 +1,5 @@
+task:
+  container:
+    image: debian:latest
+  script:
+    - printenv
```

Comment:
A newline is missing here (there are other occurrences too, marked with a red "no entry" icon).